ImageGallery/ImageGallery/ImageStore.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  DownloadStore.swift
+//  ImagaeGallery
+//
+//  Created by Nick Ryan on 6/18/24.
+//
+
+import SwiftUI
+import Combine
+import Foundation
+
+@MainActor
+class ImageStore: NSObject, ObservableObject, URLSessionDownloadDelegate {
```

Comment:
You have 3 ImageStore files but only 1 is linked in the project. Make sure to fix this